scene/animation/animation_blend_space_1d.cpp

```diff
@@ -347,18 +369,48 @@ AnimationNode::NodeTimeInfo AnimationNodeBlendSpace1D::_process(const AnimationM
 		// actually blend the animations now
 		bool first = true;
 		double max_weight = 0.0;
+
+		int mind_idx = 0;
+		Engine *eng = Engine::get_singleton();
+		double delta = eng->get_process_step() * eng->get_time_scale();
```

Comment:
```suggestion
		double delta = p_playback_info.delta;
```
Delta should be retrieved from time info to make consistent for all other AnimationNodes.

Reply:
Yep I was working on 2D the entire time and I wasn't planning on doing 1D till I had it at a point where I can pretty much transfer the changes. I forgot to make it a draft again my bad.